modules/sandboxes.am

```diff
@@ -70,14 +72,26 @@ _check_aisap() {
 			echo " OPERATION ABORTED!"
 			return 1
 		fi
-		"$AMCLIPATH" -i aisap >/dev/null 2>&1
+		if [ "$CLI" = am ] && [ -f "$APPMANCONFIG"/appman-config ]; then
+			read -r -p " ◆ DO YOU WISH TO INSTALL AISAP LOCALLY? (Y/n) " yn
+			if echo "$yn" | grep -i '^n' >/dev/null 2>&1; then
+				"$AMCLIPATH" -i aisap >/dev/null 2>&1
+			else
+				"$AMCLIPATH" -i --user aisap >/dev/null 2>&1
+			fi
+		else
+			"$AMCLIPATH" -i aisap >/dev/null 2>&1
+		fi
 		command -v aisap 1>/dev/null || return 1
 		echo " aisap installed successfully!"
-	fi	
+	fi
 	if grep "aisap-am" "$TARGET" >/dev/null 2>&1; then
 		echo " $1 is already sandboxed!"
 		return 1
 	fi
+	if [ -n "$BINDIR" ] && test -f "$BINDIR"/"$1"; then
```

Comment:
`if [ -f "$BINDIR"/"$1" ]; then`

`BINDIR` will always be set in line 24 of the main script.

Also avoid using the `test` command and use the shell `[ ]` built in which is much much faster.

Reply:
PR me